### DISCUSSION TITLE
BTV HD

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#484] [Show and tell]
### Channel ID

bTV.bg

### Stream URL (optional)

https://mx86.glebul.com/hlsfhd/hd-btv-hd_hd1080/index.m3u8?e=1692277676&hash=XGOQIq4Az2xSkHLiQgIXqA

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://mx86.glebul.com/hlsfhd/hd-btv-hd_hd1080/index.m3u8?e=1692277676&hash=XGOQIq4Az2xSkHLiQgIXqA (offline)
